src/main/java/mate/academy/service/MovieServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy.service;
+
+import java.util.Optional;
+import mate.academy.dao.MovieDao;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.Movie;
+
+@Service
+public class MovieServiceImpl implements MovieService {
+    @Inject
+    private MovieDao movieDao;
+
+    @Override
+    public Movie add(Movie movie) throws DataProcessingException {
+        return movieDao.add(movie);
+    }
+
+    @Override
+    public Movie get(Long id) throws DataProcessingException {
+        Optional<Movie> movie = movieDao.get(id);
+        if (movie.isPresent()) {
+            return movie.get();
+        }
+        throw new DataProcessingException("No movie by id: " + id);

```

Comment:
let's use NoSuchElementException here, DataProcessingException indicates that smt went wrong during request execution. In our case request was executed fine, just no value present by given id